src/EditorFeatures/CSharpTest/AddBraces/AddBracesTests.cs

```diff
@@ -15,8 +17,11 @@ public partial class AddBracesTests : AbstractCSharpDiagnosticProviderBasedUserD
         internal override (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProviderAndFixer(Workspace workspace)
             => (new CSharpAddBracesDiagnosticAnalyzer(), new CSharpAddBracesCodeFixProvider());
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddBraces)]
-        public async Task DoNotFireForIfWithBraces()
+        [Theory, Trait(Traits.Feature, Traits.Features.CodeActionsAddBraces)]
+        [InlineData((int)PreferBracesPreference.None)]
+        [InlineData((int)PreferBracesPreference.WhenMultiline)]
+        [InlineData((int)PreferBracesPreference.Always)]
```

Comment:
No clear right answer here aside from making the test class internal (which might break tests on some platforms). Will keep as is since it gives better links to the original definitions.

Reply:
Not a problem :)